lib/dry/configurable/errors.rb

```diff
@@ -7,7 +7,6 @@ module Dry
   module Configurable
     Error = Class.new(::StandardError)
 
-    AlreadyIncludedError = Class.new(Error)
```

Comment:
Hard to tell, but it may be a breaking change if somebody references this constant in their code

Reply:
Ah yes, thanks for noting that. I'll put the class back in for now, along with a comment that notes it will be deprecated soon.

Reply:
btw we have a mechanism for deprecating constants in dry-core

Reply:
Thanks again @flash-gordon, have addressed this in ca63b93 :)